src/Body/FormBody.php

```diff
@@ -164,6 +178,7 @@ private function generateMultipartFieldHeader(string $name, string $contentType)
         } else {
             $header .= "\r\n";
         }
+        $header .= "\r\n";
```

Comment:
Why did you add this line?

Reply:
Just to make the code look nicer, I removed the carrier return on the upper parts

Reply:
Oh yea, sorry, my fault xd